django_docker_engine/proxy.py

```diff
@@ -73,7 +83,11 @@ def url_patterns(self):
     def _internal_proxy_view(self, request, container_url, path_url):
         # Any dependencies on the 3rd party proxy should be contained here.
         try:
-            view = ProxyView.as_view(upstream=container_url)
+            view = ProxyView.as_view(
+                upstream=container_url,
+                add_remote_user=True)
+            if not hasattr(request, 'user'):
+                request.user = _AnonUser()
```

Comment:
@scottx611x : Do you think there's any reason to worry about modifying the request in place?